packages/core/card.ts

```diff
@@ -95,7 +95,9 @@ export class Card {
   // value for the field).
   readonly sourceCard: Card | undefined;
 
-  private readonly attributes: SingleResourceDoc['data']['attributes'];
+  // TODO: should this remain private? I needed to access the attributes in the
+  // git writer but it was private before
+  readonly attributes: SingleResourceDoc['data']['attributes'];
```

Comment:
Noooooo. please do not do this! this is definitely not what we want (and also explains why you have no linting error.

Reply:
You absolute need to use the `Card.value()` method to get values out of your card--this is wrong for so many reasons.